backend/LexBoxApi/Controllers/UserController.cs

```diff
@@ -82,6 +82,42 @@ await HttpContext.SignInAsync(user.GetPrincipal("Registration"),
         return Ok(user);
     }
 
+    [HttpGet("handleInviteLink")]
+    [RequireAudience(LexboxAudience.RegisterAccount, true)]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    public async Task<ActionResult> HandleInviteLink(LoggedInContext loggedInContext, LexBoxDbContext dbContext)
```

Comment:
Bikeshedding **welcomed** for this API name if anyone has a better idea. It needs to be different from `acceptInvitation` so that existing invite emails continue to be valid, and I didn't want it to be `acceptInvite` because that's too similar and would cause confusion.

Reply:
Actually, it *could* be acceptInvitation after all, because it'll actually be `/api/User/acceptInvitation`, which is a distinct URL from the `/acceptInvitation` page and so existing invite links will continue to work. So GET `/api/User/acceptInvitation` will be the method that is redirected to from LoginRedirect, and POST `/api/User/acceptInvitation` will be the API endpoint that the registration page submits data to.

Reply:
Changed API endpoint to `/api/User/acceptInvitation` in commit 1edf28ae.